pkg/api/upgradelog/handler.go

```diff
@@ -41,10 +41,13 @@ cd /archive/logs
 
 for f in *.log
 do
-    cat $f | awk '{$1=$2=""; print $0}' | jq -r .message > $tmpdir/logs/$f
-	if [ $? -eq 4 ]; then
-	    echo "Incomplete JSON format log line, abort processing the file $f."
-	    continue
+	cat $f | awk '{$1=$2=""; print $0}' | jq -r .message > $tmpdir/logs/$f
```

Comment:
normally, it is not necessary to `cat ` and then awk, use awk to work directly against the file 

`awk '{$1=$2=""; print $0}' $f | jq ... `

Reply:
Updated, thanks!